hasher-matcher-actioner/src/OpenMediaMatch/blueprints/matching.py

```diff
@@ -246,6 +246,9 @@ def lookup_get() -> t.Union[TMatchByBank, TBankMatchBySignalType]:
             abort(403, "Hashing is disabled, missing role")
 
         hashes = hashing.hash_media()
+        if not hashes:
+            abort(400, "Failed to hash media from URL")
```

Comment:
blocking: Hmm, in some cases this might be the expected result - if only using photo signal type, if the image is low quality, you'll get no hashes back. Hashing happened successfully and the correct result is no hashes. It's not a failure.

Can you help me understand what this is intended to catch?

Reply:
I was having some issues with hashes and figured returning a bad error over empty hashes 🤔 I guess the consumer can also handle the empty case over handling the error.